src/App.js

```diff
@@ -1,26 +1,31 @@
-import React from 'react';
-import logo from './logo.svg';
+//lab-ironbeers
+import React  from 'react';
 import './App.css';
+import { BrowserRouter as Switch, Route } from 'react-router-dom';
```

Comment:
This is wrong.

You already added `<BrowserRouter>` in the `index.js`. Also you don't want to associate it with the keyword "Switch", you want to import the `<Switch>` component so you can use it as a parent for your `<Route>` components. 

If you're unclear about how this works, please reach out to me, Thomas or Malcolm.